src/validation.h

```diff
@@ -208,7 +207,7 @@ struct MempoolAcceptResult {
     /** Constructor for failure case */
     explicit MempoolAcceptResult(TxValidationState state)
         : m_result_type(ResultType::INVALID),
-        m_state(state), m_replaced_transactions(nullopt), m_base_fees(nullopt) {
+        m_state(state), m_replaced_transactions(std::nullopt), m_base_fees(std::nullopt) {
```

Comment:
No need for these initializers. `std::optional` default initializes to `std::nullopt`